src/Icon.js

```diff
@@ -4,16 +4,22 @@ import { space, color } from 'styled-system'
 import PropTypes from 'prop-types'
 import icons from '../icons.json'
 
-// Support re-named icon names too
 // Should be removed eventually after v1.0.0
-const oldAndNewIcons = Object.assign({}, icons, {
+const aliases = {
   srollLeft: icons.chevronLeft,
```

Comment:
was the `sroll` like this like this before?

Reply:
good catch

Reply:
Looks like it